cmd/agent.go

```diff
@@ -27,9 +31,37 @@ var agentInfoCmd = &cobra.Command{
 	},
 }
 
+var agentRBACCmd = &cobra.Command{
+	Use:   "rbac",
+	Short: "print RBAC",
+	Long:  `Print RBAC string by reading GVRs`,
+	Run: func(cmd *cobra.Command, args []string) {
+
+		file, err := os.Open(agent.ConfigFilePath)
+		if err != nil {
+			log.Fatalf("Failed to load config file for agent from: %s", agent.ConfigFilePath)
+		}
+		defer file.Close()
+
+		b, err := ioutil.ReadAll(file)
+		if err != nil {
+			log.Fatalf("Failed to read config file: %s", err)
+		}
+
+		config, err := agent.ParseConfig(b)
+		if err != nil {
+			log.Fatalf("Failed to parse config file: %s", err)
```

Comment:
If you do the suggestion above then this is fine. However, worth pointing out that `log.Fatalf` uses `os.Exit` under the hood. Which causes the program to immediately terminate without handling your `defer` functions. So your `file.Close` would never actually be called.